shared/utility/module_test_utils/TestLogHandler.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2021 NUbots <nubots@nubots.net>
+ */
+
+#ifndef UTILITY_MODULETESTUTILS_TESTLOGHANDLER
+#define UTILITY_MODULETESTUTILS_TESTLOGHANDLER
+
+#include <memory>
+#include <nuclear>
+
+namespace utility::module_test {
+
+    /**
+     * @brief Handler for logging during tests so we can pass Catch the messages too
```

Comment:
Handler for logging during tests so we can pass 'Catch' messages to? So we can pass the messages to? 
pass Catch the messages too????????

Reply:
[catch](https://github.com/catchorg/Catch2) is the unit testing framework we use. So I am saying that we log the messages using catch logging macros. I could make that more clear though?

Reply:
Ok so after reading that you're wanting Catch to receive the messages as well? Defs reword since it's a bit confusing to read